### DISCUSSION TITLE
Does Cloudscape officially support/work with react 18?

### DISCUSSION THREAD

**Discussion by jebbard** [cloudscape-design/components#2069] [Q&A] [answered]
I am using react 18 for a while with Cloudscape and I am building with yarn. My application works and tests are running green. However, I was always getting a message during yarn install/build that says:

`react is listed by your project with version 18.2.0, which doesn't satisfy what @cloudscape-design/components (p1eae0) and other dependencies request (^16.8.0 || ^17.0.0)`

When migrating from yarn 1.x to yarn 4.x this turns into an error.

Does it mean Cloudscape requires react 17 and does not officially support react 18 yet?

**Comment by michaeldowseza** [comment, marked as answer]
Hello!

From the components point of view React 18 shouldn't be a problem as showed by the [peerDependencies](https://github.com/cloudscape-design/components/blob/main/package.json#L50-L51) for the package. We also use React 18 with the components in our [board-components package](https://github.com/cloudscape-design/board-components/blob/main/package.json#L56). You might need to provide the full log message for me to debug but its likely that this error might be coming from a different dependency within your package.

Kind regards,
Michael
